content/rs/administering/security/account-management.md

```diff
@@ -5,62 +5,163 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-You can view and update the cluster users in the cluster **Settings > team** page.
-
-## User Roles
-
-The user roles included in Role Based Access Control (RBAC) are:
-
-|  **Role** | **Description** | **Details** |
-|  ------ | ------ | ------ |
-|  **Admin** | Has full access to the system |  |
-|  **DB Viewer** | Allowed to view DB configuration/metrics. All Node/Cluster information and settings are unavailable | Can view info about all databases on the cluster<br/><br/>Cannot view info about nodes and cluster<br/><br/>Can view logs<br/><br/>Cannot view cluster settings outside of changing own password |
-|  **Cluster Viewer** | Allowed to view Cluster and DB configuration/metrics. | Can view all info about Cluster, nodes and databases.<br/><br/>Can view logs<br/><br/>Cannot view cluster settings outside of changing own password |
-|  **DB Member** | Allowed to view and edit DB configuration. All Node/Cluster information and settings are unavailable | Can create databases<br/><br/>Can view db metrics<br/><br/>Can edit database configurations<br/><br/>Can clear slowlog<br/><br/>Can view logs<br/><br/>Cannot view info about nodes and cluster<br/><br/>Cannot view cluster settings outside of changing own password |
-|  **Cluster Member** | Allowed to view Node/DB information and edit DB configurations | Can view info about nodes and cluster<br/><br/>Can create databases<br/><br/>Can view db metrics<br/><br/>Can edit database configurations<br/><br/>Can clear slowlog<br/><br/>Can view logs<br/><br/>Cannot view cluster settings outside of changing own password |
-
-All roles apply to both the UI and API levels.
-
-You can assign users with these roles through Settings -\> team page in
-the UI or through the users API.
-
-## Adding a User
-
-**To add a user:**
-
-1. Click the + (plus) sign at the bottom of the table.
-1. Enter the name, email and password of the new user.
-1. Select which Role the user should have
-1. Indicate whether the user will receive email alerts.
-1. Click **Save** icon.
-
-    ![useradd](/images/rs/useradd-300x101.png)
-
-## Updating a User
-
-**To update a user:**
-
-1. Click the **Edit** icon at the far right of the user row.
-1. Edit the user details or delete the user.
+To give each user only the permissions that they need for their work with the cluster,
+RS lets you assign a role to each user.
+You can manage users and roles in **settings** > **team**, or with the REST API.
+
+The roles and permissions available in RS are:
+
+<table>
+  <tr>
+    <th></th>
+    <th colspan="5">Database</th>
+    <th colspan="3">Nodes</th>
+    <th colspan="5">Cluster</th>
+  </tr>
+  <tr>
+    <td align="center"></td>
+    <td class="cat-boundary" align="center">View metrics</td>
+    <td align="center">View<br>config</td>
+    <td align="center">View<br>redis<br>password</td>
+    <td align="center">Edit config</td>
```

Comment:
(I'm not sure this is the correct line to comment on, but-) No one can edit cluster / node configuration from the UI.

Reply:
Ok. I'll remove the Edit config from node and cluster